### ISSUE TITLE
Document code

### ISSUE THREAD

**Issue by ericholiveira** [ericholiveira/studio#48]
We must document studio code and use some lib to generate documentation from it. Any ideas on the lib?

**Comment by avishnyak** [comment]
I tend to use [DocStrap](https://github.com/docstrap/docstrap) (a JSDoc3-based doc generator) because it's easy to get running, everyone understands [JSDoc](http://usejsdoc.org), is themeable, has search built in and has the ability to add markdown tutorials right to the final output.

Here is an example including tutorials:
http://docstrap.github.io/docstrap/themes/flatly/index.html

**Comment by sean-hill** [comment]
@ericholiveira I really like [gitbook](https://www.gitbook.com/). It's really sexy and simple to use. Check out [featherjs](http://docs.feathersjs.com/) as an example. It's also easy to use with Github, so that's a plus 👍

**Comment by sean-hill** [comment]
@ericholiveira we could also create a Studio organization because it sounds like we'll have multiple repositories, and it'd be helpful to not have them all under your own github account, and have us all as contributors instead.

**Comment by sean-hill** [comment]
@ericholiveira as an example I threw this together really quick.

[Gitbook example](https://sean-hill.gitbooks.io/studio/content/index.html)

**Comment by ericholiveira** [comment]
@sean-hill i completely agree with a Studio organization, thats the reason i`ve been asking you guys about your vision on the future of the project to make sure we are all on the same page.
Also, i loved this gitbook, its possible to generate api documentation from code`s comments like jsdoc? If its not possible, can we integrate both (jsdoc and gitbook)?

**Comment by avishnyak** [comment]
@ericholiveira we can probably make jsdoc to gitbook happen by using (jsdoc-to-markdown)[https://www.npmjs.com/package/jsdoc-to-markdown] as part of the build steps...
